### DISCUSSION TITLE
MX Master 3s middle button

### DISCUSSION THREAD

**Discussion by ptrunley** [noah-nuebling/mac-mouse-fix#927] [Questions]
Is it possible to remap the MX Master 3s middle button? It does not show up when I click the crosshairs and instead is always set as the scroll wheel mode switch button. I'd prefer to use it as a general purpose middle button. Thanks.

**Comment by willisblackburn** [comment]
+1.

I'm concerned about the lack of response from the developer to most of the questions posted here. I found the app and thought I'd finally be able to get rid of the awful Logi Options+ software. But maybe not.
